src/quo/design_system/colors.cljs

```diff
@@ -0,0 +1,50 @@
+(ns quo.design-system.colors
+  (:require [reagent.core :as reagent]))
+
+(def white "#FFFFFF")
+(def black "#000000")
+
+(def light-theme
+  {:positive-01    "rgba(68,208,88,1)"     ; Primary Positive, text, icons color
```

Comment:
im not sure i understand these names and values it kinda hard to match with figma

Reply:
Added here some mapping of colors from figma with the code https://github.com/status-im/status-react/pull/10338/files#diff-27ee3b73723c60dff177c9746948ca4dR7 wdyt?